core/drivers/stm32_cpu_opp.c

```diff
@@ -266,6 +276,37 @@ static TEE_Result stm32_cpu_opp_is_supported(const void *fdt, int subnode)
 	return TEE_SUCCESS;
 }
 
+static TEE_Result cpu_opp_pm(enum pm_op op, unsigned int pm_hint,
+			     const struct pm_callback_handle *hdl __unused)
+{
+	assert(op == PM_OP_SUSPEND || op == PM_OP_RESUME);
+
+	/* nothing to do if OPP is managed by Linux and not by SCMI */
+	if (!IS_ENABLED(CFG_SCMI_MSG_PERF_DOMAIN))
+		return TEE_SUCCESS;
```

Comment:
Prefer no call `register_pm_driver_cb()` if build config tells the PM sequences are not needed.